src/cmd.rs

```diff
@@ -427,6 +427,13 @@ pub enum AdminSub {
         #[structopt(subcommand)]
         target_type: DeleteSub,
     },
+
+    /// [WIP] Create or update DynamoDB tables based on CloudFormation template files (.cfn.yml).
+    #[structopt(setting(structopt::clap::AppSettings::Hidden))]
+    Apply {
+        #[structopt(long)]
```

Comment:
Could you add a code comment to describe the intention of this option? Due to no comment, the current `cargo run admin apply --help` does not provide any information regarding `--dev` option.

```bash
❯ cargo run admin apply --help
    Finished dev [unoptimized + debuginfo] target(s) in 0.32s
     Running `target/debug/dy admin apply --help`
dy-admin-apply 0.2.1
[WIP] Create or update DynamoDB tables based on CloudFormation template files (.cfn.yml)

USAGE:
    dy admin apply [FLAGS] [OPTIONS]

FLAGS:
        --dev        
    -h, --help       Prints help information
    -V, --version    Prints version information

OPTIONS:
    -p, --port <port>        Specify the port number. This option has an effect only when `--region local` is used
    -r, --region <region>    The region to use (e.g. --region us-east-1). When using DynamodB Local, use `--region
                             local`. You can use --region option in both top-level and subcommand-level
    -t, --table <table>      Target table of the operation. You can use --table option in both top-level and subcommand-
                             level. You can store table schema locally by executing `$ dy use`, after that
                             you need not to specify --table on every command
```

Reply:
add at https://github.com/awslabs/dynein/pull/156/commits/226bc1c21528e366b8245e47638b18880b1224ae